File: JAVA_CalculadoraReflection/src/br/ifes/projetosistemas/calculadora/dto/RequestDTO.java
package br.ifes.projetosistemas.calculadora.dto;

public class RequestDTO {
    private int opcao;
    private int valor1;
    private int valor2;

    public RequestDTO(int opcao, int valor1, int valor2) {
        this.opcao = opcao;
        this.valor1 = valor1;
        this.valor2 = valor2;
    }

    public int getOpcao() {
        return opcao;
    }

    public int getValor1() {
        return valor1;
    }

    public int getValor2() {
        return valor2;
    }
}

File: CSharp_Calculadora/CalculadoraConsole/DTOs/Request/RequestDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CalculadoraConsole.Application.DTOs.Request
{
    public class RequestDTO
    {
        public int Opcao;
        public int Valor1;
        public int Valor2;

        public RequestDTO(int Opcao, int Valor1, int Valor2) 
        { 
            this.Opcao = Opcao;
            this.Valor1 = Valor1;
            this.Valor2 = Valor2;
        }

        public int getOpcao()
        {
            return Opcao;
        }

        public int getValor1()
        {
            return Valor1;
        }

        public int getValor2()
        {
            return Valor2;
        }
    }
}
